sass/_theme_rst.sass

```diff
@@ -26,10 +26,30 @@
     margin-bottom: $base-line-height
     p.caption
       font-style: italic
+      font-family: inherit !important
+      font-size: inherit !important
+      color: $caption-color !important
```

Comment:
We should never have to use `!important`. Specifying a selector with great specificity makes it more explicit what we're changing, and reduces the problems with overriding that `!important` introduces. A number of folks extend the theme and this would cause problems.